### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by mrdrogdrog**
### Component/Part
Github pull request template

### Description
This PR adds a template for github pull requests

### Steps

- [x] added implementation
- [x] added / updated tests
- [x] added / updated documentation
- [x] extended changelog

### Related Issue(s)
#70

### CODE REVIEW

**Comment by ErikMichelson** [comment]
In our issue templates the example values are given as comments. Maybe in the PR template that should be similar.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### Component/Part
+AwesomeComponent37
+
+### Description
+This PR fixes/adds/improves/...
+
+### Related Issue(s)
+#123
```
Comment (DerMolly) [inline .github/pull_request_template.md:8]:
```suggestion
<!-- e.g #123 -->
```

**Comment by SISheogorath** [comment]
It might be a good idea to add a checklist:

- [ ] docs
- [ ] changes
- [ ] tests
- [ ] changelog

This tends to improve the documentation of features a lot.

**Comment by DerMolly** [comment]
@SISheogorath What's the difference between changes and changelog?

**Comment by SISheogorath** [comment]
Changes are your code changes, that actually change your bahavior of the app (there is probably a way better name for it), the changelog is more of a documentation task.

**Comment by SISheogorath** [comment]
Just checked, in the project where I already use such a checklist in the PR template it's called `implementation` instead of `changes`. That should be more on the spot :)

**Comment by DerMolly** [comment]
so with `ìmplementation` you mean the PR contains the actual code changes?

**Comment by SISheogorath** [comment]
Yes. It's mainly there for completeness but also to make sure people point out why maybe code changes are not needed (as with bug fixes that might don't need any changes of documentation).

It helps to provide some additional context to each PR. And of course it's a self-check for the person creating the PR so no files were forgot to add during the commit.

**Comment by DerMolly** [comment]
Since people are only required to tick a checkbox I think we should elaborate a bit more and not just use one word abreviations.
```
### Steps

please tick steps this PR performs (if something is not necessary, please tick anyway to indicate you considered it)

- [ ] added implementation
- [ ] added / updated tests
- [ ] added / updated documentation
- [ ] extended changelog
```

**Comment by mrdrogdrog** [comment]
I like the check list. Maybe we could put the second part into a comment? It's not necessary, except if you're editing the PR.

**Pull request merged**